### ISSUE TITLE
Audio playback time inconsistency after jumping to a specific timestamp

### ISSUE THREAD

**Issue by aliciusschroeder** [CR1337/rl-audio-player#7]
I encountered an issue when jumping to different positions in an audio file during playback. The reported current time seems to be inconsistent with the expected position after jumping.

**Steps to reproduce:**
1. Create a WAV file (```sox -n -R -r 41000 -b 16 -c 2 white_noise_2c_64s.wav synth 64 whitenoise```)
2. Load the audio file (`./build/main test_data/white_noise_2c_64s.wav`)
3. Playback starts automatically
4. Check the current time a few times using the `t` command
5. Pause playback with `p`
6. Jump to 50 seconds with `j 50000`
7. Check the current time with `t` - observe it reports around 5.22 seconds instead of the expected 50 seconds
8. Resume playback with `r`, then pause again with `p`
9. Check the current time, jump to 2 seconds with `j 2000`, then check time again
10. Jump to 10 seconds with `j 10000` and check time - observe it reports 40 seconds

**Expected behavior:**
After jumping to a specific timestamp with the `j` command, the `t` command should report the current time as close to the jumped position when entered quickly (<500 ms) after jumping. Subsequent playback should advance from that time.

**Actual behavior:**
The current time reported by `t` seems inconsistent and incorrect after jumping to various positions, not matching the timestamp that was jumped to. Playback after jumping also starts from an unexpected time rather than the selected position.

**Console Output** (t being entered quickly after jumping):
```
$ sox -n -R -r 41000 -b 16 -c 2 test_data/white_noise_2c_64s.wav synth 64 whitenoise
$ ./build/main test_data/white_noise_2c_64s.wav
Total duration: 64.00 seconds
Playing
t
Current time: 3.42 seconds
t
Current time: 6.04 seconds
p
Paused
j 50000
Jumped to 50000 milliseconds
t
Current time: 5.22 seconds
r
Resumed
p
Paused
t
Current time: 7.30 seconds
j 2000
Jumped to 2000 milliseconds
t
Current time: 8.00 seconds
j 10000
Jumped to 10000 milliseconds
t
Current time: 40.00 seconds
```

The issue seems to be reproducable with many different sorts of audio file

**Comment by CR1337** [comment]
I guess this also is related to the other "time" issues. I will have a look at it.

**Comment by CR1337** [comment]
Fixed with #10
